app/screens/AppNavigator/screens/Transactions/TransactionsScreen.tsx

```diff
@@ -47,21 +51,50 @@ export function TransactionsScreen (): JSX.Element {
 
   useEffect(() => loadData(), [])
 
+  if (isEmpty) {
+    return (
+      EmptyTransaction(navigation)
+    )
+  } else {
+    return (
+      <FlatList
+        testID='transactions_screen_list'
+        style={tailwind('w-full')}
+        data={activities}
+        renderItem={TransactionRow(navigation)}
+        keyExtractor={(item) => item.id}
+        ListFooterComponent={hasNext ? LoadMore(loadData) : undefined}
+        refreshControl={
+          <RefreshControl
+            refreshing={loadingStatus === 'loading'}
+            onRefresh={loadData}
+          />
+        }
+      />
+    )
+  }
+}
+
+function EmptyTransaction (navigation: NavigationProp<TransactionsParamList>): JSX.Element {
   return (
-    <FlatList
-      testID='transactions_screen_list'
-      style={tailwind('w-full')}
-      data={activities}
-      renderItem={TransactionRow(navigation)}
-      keyExtractor={(item) => item.id}
-      ListFooterComponent={hasNext ? LoadMore(loadData) : undefined}
-      refreshControl={
-        <RefreshControl
-          refreshing={loadingStatus === 'loading'}
-          onRefresh={loadData}
-        />
-      }
-    />
+    <View
+      testID='empty_transaction'
+      style={tailwind('px-8 pt-32 pb-2 text-center')}
+    >
+      <MaterialCommunityIcons name='folder-alert' size={44} color='#212121' style={tailwind('pb-5 text-center')} />
```

Comment:
use tailwind style instead of color.

Reply:
should I add the custom color into tailwind config, or to ask designer to replace it with a predefined tailwind color?

Reply:
you can use `text-black`